src/screens/AddSiteScreen.js

```diff
@@ -3,10 +3,34 @@ import { Alert, Picker, ScrollView, Modal, View} from 'react-native';
 
 import { Button, FormLabel, FormInput, Input, Icon, Overlay, Text } from 'react-native-elements';
 
-import {campsite} from '../locale.en';
-import {navyBlue, grey, darkBlue, lightwhiteblue} from '../styles/index';
+import {campsite, campsite_form, formFeedback, facilities_options, accessibility_options, submit_form, common} from '../locale.en';
 
-import {CardSection, Card, largeFormElement} from '../components/common/index';
+const{site_description, upload} = campsite
+
+const {latitude, longitude,
+       longitude_placeholder, latitude_placeholder,
+       add_site_title,site_info,
+       description,description_placeholder,
+       directions,directions_placeholder,
+       nearest_town,nearest_town_placeholder,
+       here_now, add_site } = campsite_form;
+
+const { full_service, some,
+        none, permit,
+        paid, free,
+        facilities } = facilities_options;
```

Comment:
So here in your input you could destructure the 'campsite_form' and  `facilities_options` that you need from your `campsite` import with nested destructuring. Something like:

```
 const {site_description, upload, campsite_form: {longitude, latitude, accessibility_options:{accessibility, paved_road}})
```